engine/apps/alerts/tasks/notify_user.py

```diff
@@ -528,7 +528,7 @@ def perform_notification(log_record_pk, use_default_notification_policy_fallback
                 )
                 UserNotificationPolicyLogRecord(
                     author=user,
-                    type=UserNotificationPolicyLogRecord.TYPE_PERSONAL_NOTIFICATION_FAILED,
+                    type=UserNotificationPolicyLogRecord.TYPE_PERSONAL_NOTIFICATION_SUCCESS,
```

Comment:
FWIW, for the notification auditor SUCCESS or FAILED are equally good (just in case).